lib/litani_report.py

```diff
@@ -93,8 +99,9 @@ def render_to_file(self, out_file):
         return not proc.returncode
 
 
-    @staticmethod
-    def render(render_root, pipe_url, pipe):
+    def render(self, render_root, pipe_url, pipe):
```

Comment:
The `render_to_file` method above no longer needs to wrap `dot` in a try-catch because you're already checking that `dot` is installed in the constructor.